zng/cmake.rs

```diff
@@ -14,6 +14,9 @@ pub fn build_zlib_ng(target: &str, compat: bool) {
             .define("WITH_DFLTCC_INFLATE", "1")
             .cflag("-DDFLTCC_LEVEL_MASK=0x7e");
     }
+    if target.contains("riscv64") && (cfg!(feature = "rvv-off") || env::var_os("RVV_OFF").is_some()) {
```

Comment:
Generally I prefer "positive" features over "negative" features. By this I mean that the feature sounds like it adds something rather than taking something away.

For example, you could name the environment variable `WITH_RVV` and allowing the values `ON` and `OFF`. This would be familiar to someone who knows the `zlib-ng` build. It would allow forcing the use of RVV if there was ever a case where it was disabled.

You could name the feature `with-rvv`. However this would require the default to compile without RVV. This has the advantage of working in more places, at the cost of behaving differently to the default behavior of `zlib-ng`.